modules/custom/az_paragraphs/src/Plugin/paragraphs/Behavior/AZLinkGroupParagraphBehavior.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+namespace Drupal\az_paragraphs\Plugin\paragraphs\Behavior;
+
+use Drupal\Component\Utility\Html;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\paragraphs\ParagraphInterface;
+
+/**
+ * Provides a behavior for text with background.
+ *
+ * @ParagraphsBehavior(
+ *   id = "az_link_group",
+ *   label = @Translation("Quickstart Link Group Paragraph Behavior"),
+ *   description = @Translation("Provides class selection for link group."),
+ *   weight = 0
+ * )
+ */
+class AZLinkGroupParagraphBehavior extends AZDefaultParagraphsBehavior {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildBehaviorForm(ParagraphInterface $paragraph, array &$form, FormStateInterface $form_state) {
+    $config = $this->getSettings($paragraph);
+    $linkgroup_style_unique_id = Html::getUniqueId('');
+
+    // Link Group style.
+    $form['group_style'] = [
+      '#title' => $this->t('Style'),
+      '#type' => 'select',
+      '#default_value' => $config['group_style'] ?? 'buttons',
+      '#description' => $this->t('Determines the style/behavior of your group of links.'),
+      '#options' => [
+        'buttons' => $this->t('Buttons'),
+        'dropdown' => $this->t('Dropdown'),
+        'list_group' => $this->t('List Group'),
+      ],
+      '#attributes' => [
+        'id' => $linkgroup_style_unique_id,
+      ],
+    ];
+
+    // Dropdown title.
+    $form['dropdown_title'] = [
+      '#title' => $this->t('Dropdown Title'),
+      '#type' => 'textfield',
+      '#description' => $this->t('The title of your dropdown menu.'),
+      '#size' => 60,
+      '#maxlength' => 120,
+      '#default_value' => $config['dropdown_title'] ?? '',
+      '#states' => [
+        'visible' => [
+          ':input[id="' . $linkgroup_style_unique_id . '"]' => ['value' => 'dropdown'],
+        ],
+        'required' => [
+          ':input[id="' . $linkgroup_style_unique_id . '"]' => ['value' => 'dropdown'],
+        ],
+      ],
+    ];
+
+    // Button color.
+    $form['button_color'] = [
+      '#title' => $this->t('Button Color'),
```

Comment:
```suggestion
      '#title' => $this->t('Dropdown Button Color'),
```

Reply:
This would change the title for regular buttons as well. I think it's best to keep it as "button color" so it stays applicable to both dropdowns and buttons.